packages/api-kit/tests/endpoint/index.test.ts

```diff
@@ -645,6 +645,77 @@ describe('Endpoint tests', () => {
         }
       })
     })
+
+    it('getSafeOperationsByAddress', async () => {
+      await chai
+        .expect(safeApiKit.getSafeOperationsByAddress(safeAddress))
+        .to.be.eventually.deep.equals({ data: { success: true } })
+      chai.expect(fetchData).to.have.been.calledWith({
+        url: `${txServiceBaseUrl}/v1/safes/${safeAddress}/safe-operations/`,
+        method: 'get'
+      })
+    })
+
+    it('getSafeOperation', async () => {
+      const safeOperationHash = 'safe-operation-hash'
+
+      await chai
+        .expect(safeApiKit.getSafeOperation(safeOperationHash))
+        .to.be.eventually.deep.equals({ data: { success: true } })
+      chai.expect(fetchData).to.have.been.calledWith({
+        url: `${txServiceBaseUrl}/v1/safe-operations/${safeOperationHash}/`,
+        method: 'get'
+      })
+    })
+
+    it('addSafeOperation', async () => {
+      const signature = '0xsignature'
+      const moduleAddress = '0xa581c4A4DB7175302464fF3C06380BC3270b4037'
+
+      const safeOperation = {
+        data: {
+          nonce: 42,
+          initCode: '0x123initcode',
```

Comment:
I know this is mocked but it might have been cool to use structurally valid data. I think in this case this would have failed with a bad request since their serializer expect to be a valid hex string.